src/components/containers/Header.js

```diff
@@ -10,27 +11,19 @@ export default class Header extends Component {
     render (){
 
         const buttons = [
-            <FlatButton>
-                <a href="/AdminPage">Services</a>
-            </FlatButton>,
-            <FlatButton>
-                Places
-            </FlatButton>,
-            <FlatButton>
-                <a href="/ServiceForm">New Service</a>
-            </FlatButton>
+            <FlatButton  hoverColor='transparent' rippleColor='transparent' labelStyle={{color: 'white'}} href="/AdminPage" label="Service" />,
```

Comment:
FlatButton should not have a href field, but should be embeded in a react-router-dom Link element

Reply:
![image](https://user-images.githubusercontent.com/21339793/37348156-bb41efa2-26d3-11e8-8a03-140d23c89c42.png)